apps/projects/app/components/Card/Issue.js

```diff
@@ -21,6 +21,7 @@ const labelsTags = (labels, theme) =>
       css="margin-right: 10px; width: auto"
       background={'#' + label.node.color + '99'}
       color={`${theme.surfaceContent}`}
+      uppercase={false}
```

Comment:
Just as a side comment, I think omitting the prop would be perfectly fine on boolean props to denote a false value on them.
```suggestion
```

Reply:
Yes, but the default is `true`, so I need to explicitly give it `false` value.